bin/tilelive-copy

```diff
@@ -27,7 +27,7 @@ if (!argv._[0]) {
     console.log('Options:');
     console.log('  --scheme=[scanline,pyramid,list]  Default: scanline.');
     console.log('  --list=[filepath]                 Filepath if scheme is list.');
-    console.log('  --concurrency=[number]            Copy concurrency.');
+    console.log('  --concurrency=[number]            Control on the number of pending I/O operations with the underlying source during copy (this is not CPU concurrency)');
```

Comment:
👍

Reply:
Perhaps you could also include an additional note here to the effect that "CPU concurrency is handled at the plugin level. Some support the `UV_THREADPOOL_SIZE ` environment variable, but you'll need to check the plugin(s) you are using for more details." This might not be the spot for it since that's quite verbose for a command line doc, but I guess there's not really any manpage or other longer format doc...

Reply:
👍 great idea @ianthetechie - I think since most people will likely interface with this main module's documentation first, it makes sense to put some general tips and information here that may help others as they dive into specific plugins.